### ISSUE TITLE
opam package does not build with latest mathcomp-finmap

### ISSUE THREAD

**Issue by Armael** [math-comp/analysis#94]
Trying to install the library using the opam package currently does not work, because of finmap it seems (it works if I pin finmap 1.0.0 instead of using finmap dev).

The log:
```
make -f Makefile.coq
make[1] : on entre dans le répertoire « /home/armael/.opam/current/.opam-switch/build/coq-mathcomp-analysis.dev »
COQDEP VFILES
COQC altreals/xfinmap.v
COQC boolp.v
COQC forms.v
COQC reals.v
COQC classical_sets.v
File "./altreals/xfinmap.v", line 83, characters 30-31:
Error:
In environment
R : Type
idx : R
op : Monoid.com_law 1
T : choiceType
s : seq T
P : ?T
F : ?T0
The term "s" has type "seq T" while it is expected to have type "unit".

make[2]: *** [Makefile.coq:657: altreals/xfinmap.vo] Error 1
```

**Comment by affeldt-aist** [comment]
> Trying to install the library using the opam package currently does not work,
> because of finmap it seems (it works if I pin finmap 1.0.0 instead of using finmap dev).

Indeed the version in master at the time of this writing is using finmap 1.0 and
README.md does not reflect this. This compatibility issue is due to recent changes
in finmap.

**Comment by CohenCyril** [comment]
Closed by #96
